web/styles/settings.css

```diff
@@ -1,8 +1,6 @@
 :root {
-    /* The height of the settings header (including tab switcher). */
-    --settings-header-height: 45px;
     /* The width of the settings sidebar. */
-    --settings-sidebar-width: 255px;
+    --settings-sidebar-width: 18.2142em; /* 255px at 14px/em */
```

Comment:
Is there a reason this variable can't live with the others in `app_variables.css`?

Reply:
Mostly I didn't move it because I didn't have to and was minimizing changes. I can add another commit moving it.